functional-samples/sample.co2meter/main-page.html

```diff
@@ -0,0 +1,19 @@
+<html>
+  <head>
+    <title>CO2 Meter</title>
+    <script type="module" src="main-page.js"></script>
+    <style>
+      iframe {
+        border: none;
+      }
+    </style>
+  </head>
+  <body>
+    <div>
+      <button type="button" id="grantPermissionButton">
```

Comment:
Would it be easy to hide this button if the permission has already been granted?

Reply:
We don't have any plan to integrate that yet and. [navigator.hid.getDevices](https://wicg.github.io/webhid/#getdevices-method) does document that this `getDevices` method can be used for retrieving permitted devices.